GenumKit/Stencil/Contexts.swift

```diff
@@ -83,13 +85,20 @@ extension StoryboardParser {
     let storyboards = Set(storyboardsScenes.keys).union(storyboardsSegues.keys).sort(<)
     let storyboardsMap = storyboards.map { (storyboardName: String) -> [String:AnyObject] in
       var sbMap: [String:AnyObject] = ["name": storyboardName]
+      // Initial Scene
       if let initialScene = initialScenes[storyboardName] {
-        var initial: [String:AnyObject] = ["viewController": initialScene.customClass ?? "UI\(uppercaseFirst(initialScene.tag))"]
```

Comment:
That was the problematic line, because we don't want the SwiftGen binary itself to add the `UI` prefix (so that we can also be compatible with OS X and its NS prefix) but want that to be handled at the template level (so that OS X templates could add the `NS` prefix)